### DISCUSSION TITLE
Filtering Supabase

### DISCUSSION THREAD

**Discussion by jorgecamus24** [supabase/supabase#14296] [Questions]
Hello, I am making a filter to find a record ( imei = 830675132116850), this filter as a GET method returns the entire table, what am I doing wrong. 
This is the code that I put in Postman raw:

let { data: validacion_imei, error } = await supabase
  .from('validacion_imei')
  .select("imei")
  .filter('imei', 'eq', '830675132116850')

**Comment by GaryAustin1** [comment]
It is unclear what you are doing.
The code you show is javascript to be used in the javascript client, but you mention using GET and Postman which would  not take javascript.

**Comment by jorgecamus24** [comment]
Hello Gray Thank you very much for your support, the problem is already
solved.

I take the opportunity for the following requests:

- Supabase supports 200 million records and filtering processing in such a
way that a table with UID and IMEI records is compared with another table
that only has IMIE and reports all IMEIs it finds that have no UID.

I am working with Bubble, if possible, tell me how I can make this
application.

Finally, I would like to know the monthly cost of handling 50 million
emails and 300 million registrations.

I remain to the comments

Greetings
Jorge Camus

**Reply by GaryAustin1** [reply]
You are better off starting new questions on different  topics so you can get help.
I don't know anything about Bubble and other users who might help are not likely to associate your first question and title with Bubble.

As for you pricing questions.  
Emails would be handled by your SMTP provider which can be setup on the instance.  Supabase built in email is more for testing or very small projects.  
300M registrations using Auth is alot.   There is no direct cost for the registration but the pricing page https://supabase.com/pricing shows a cost for each unique active user each month.  That level would seem to require an enterprise level plan.  I am not an employee and you probably need to reach out here https://app.supabase.com/support/new to inquire on pricing if the pricing page does not answer your question.
